user-journeys/application_developer-foundational.html

```diff
@@ -16,80 +16,112 @@
-          <a href="#1"><div class="docButton">1.&nbsp;Setup&nbsp;a&nbsp;development&nbsp;environment</div></a>
+          <a href="#1"><div class="docButton">1.&nbsp;Set up&nbsp;a&nbsp;development&nbsp;environment</div></a>
           <a href="#2"><div class="docButton">2.&nbsp;Deploy,&nbsp;scale,&nbsp;and&nbsp;update&nbsp;an&nbsp;application</div></a>
-          <a href="#3"><div class="docButton">3.&nbsp;Understand&nbsp;Kubernetes&nbsp;basics</div></a>
+          <a href="#3"><div class="docButton">3.&nbsp;Understand&nbsp;basic&nbsp;Kubernetes&nbsp;objects</div></a>
           <a href="#4"><div class="docButton">4.&nbsp;Additional&nbsp;resources</div></a>
       </div>
       <div class="docsection1">
 
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean id felis non enim faucibus lacinia. Aliquam massa mauris, interdum a ex ut, sagittis rutrum nulla. In pellentesque est at molestie fringilla. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec eu mi at velit lacinia venenatis ac nec sem. In volutpat pellentesque dui ut commodo. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Duis sollicitudin eleifend felis non facilisis. Pellentesque leo urna, congue id auctor non, varius a nunc. Duis ultrices, odio ut hendrerit suscipit, nisi mauris dignissim mauris, nec bibendum ante neque ut augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam condimentum libero sit amet rutrum fermentum. Proin posuere condimentum odio. Aenean volutpat, ex vel tincidunt tincidunt, massa odio condimentum lectus, vel iaculis libero dolor in nisi.
+        This page and its linked topics introduce you to foundational concepts and tasks for getting started with Kubernetes if you're an application developer. The focus is on a development workflow -- check out other user journeys to learn about managing production clusters, or providing high availability (HA) for critical application deployments.
+<br><br>
+        We start with hands-on tasks -- how to install and configure a simple Kubernetes cluster on your local machine, and how to deploy and manage applications on your new cluster. If you'd rather read more about Kubernetes terms and concepts first, you can skip to <a href="#3">Understand Kubernetes basics</a>.
+<br><br>
+        TODO: Add link back to setup from basics, depending on how much text we wind up with and what other navigation is available on the page
+<br><br>
+        TODO: decide how much intro conceptual material to put here. Working from <a href="https://kubernetes.io/docs/tutorials/kubernetes-basics/cluster-intro/">Using Minikube to Create a Cluster</a> here; see Kubernetes basics section for a different way of presenting the same information. TO DISCUSS: Large amount of near-duplicate material in different sections of the docs. Edit and move here? Edit in place and link? How much do we want to normalize/reconcile and remove?
+<br><br>
```

Comment:
Let's rethink the entire body of minikube material. Get rid of the old, and replace with material that's consolidated and simple.